cli/waiter/subcommands/create.py

```diff
@@ -0,0 +1,131 @@
+import argparse
+import logging
+
+import requests
+
+from waiter import terminal, http
+from waiter.querying import get_token
+from waiter.util import guard_no_cluster, print_info
+
+TRUE_STRINGS = ('yes', 'true')
+FALSE_STRINGS = ('no', 'false')
+BOOL_STRINGS = TRUE_STRINGS + FALSE_STRINGS
+
+create_parser = None
+
+
+def print_post_result(resp):
+    """Prints the result of a token POST"""
+    resp_json = resp.json()
+    if 'message' in resp_json:
+        message = resp_json['message']
+        print_info(f'{message}.')
+        return
+
+    if 'waiter-error' in resp_json and 'message' in resp_json['waiter-error']:
+        message = resp_json['waiter-error']['message']
+    else:
+        message = 'Encountered unexpected error'
+    raise Exception(f'{message}.')
+
+
+def post_failed_message(cluster_name, reason):
+    """Generates a failed token post message with the given cluster name and reason"""
+    return f'Token post {terminal.failed("failed")} on {cluster_name}:\n{terminal.reason(reason)}'
+
+
+def create_or_update(cluster, token_name, token_fields):
+    """Creates (or updates) the given token on the given cluster"""
+    cluster_name = cluster['name']
+    cluster_url = cluster['url']
+
+    token = get_token(cluster, token_name)
+    try:
+        print_info('Attempting to post on %s cluster...' % terminal.bold(cluster_name))
+        json_body = token if token else {}
+        json_body.update(token_fields)
+        resp = http.post(cluster, 'token', json_body, params={'token': token_name})
```

Comment:
- Should we include the etag here on the post? 
- We should also include the accept: application/json header to request Waiter to return the response as json.

Reply:
Added the `If-Match` to the post

Reply:
Added the `Accept: application/json` to the post as well